Sources/SQLiteNIO/SQLiteConnection.swift

```diff
@@ -282,3 +221,11 @@ public final class SQLiteConnection: SQLiteDatabase {
         assert(self.handle == nil, "SQLiteConnection was not closed before deinitializing")
     }
 }
+
+final class UnsafeMutableTransferBox<Wrapped>: @unchecked Sendable {
+    var wrappedValue: Wrapped
+    init(_ wrappedValue: Wrapped) { self.wrappedValue = wrappedValue }
+}
+
+extension SQLiteConnection.Storage: Sendable {}
+extension SQLiteConnection: @unchecked Sendable {}
```

Comment:
I'd rather have the pointer wrapped in `UnsafeMutableTransferBox` or something else rather than disabling checking across the whole type

Reply:
Ended up dealing with it by utility-wrapping the `runIfActive()` calls (since captures of `self` were, aside from one the only complaints without the `@unchecked`). The one other complaint (the mutability of `handle` I dealt with by giving it its own version of a mutable transfer box.